.gitignore

```diff
@@ -17,4 +18,4 @@ coverage
 # editors
 .idea/*
 *.iml
-*.sublime-*
\ No newline at end of file
+*.sublime-*
```

Comment:
missed this. I don't think we should have an editors section. `gitignore` is for artifacts of the _project itself_, not user systems